src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSMarshalerType.cs

```diff
@@ -37,47 +38,53 @@ private JSMarshalerType(JSFunctionBinding.JSBindingType signatureType)
         /// <summary>
         /// Marshaling ignores result of the JavaScript function.
         /// </summary>
+        /// <returns>The marshaler metadata.</returns>
         public static JSMarshalerType Discard { get; } = new JSMarshalerType(new JSFunctionBinding.JSBindingType
         {
             Type = MarshalerType.Discard
         });
 
         /// <summary>
-        /// Marshal as JavaScript <see href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean">Boolean</see> type.
+        /// Marshal as JavaScript <see href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean">Boolean.</see> type.
```

Comment:
Was this dot intended?

```suggestion
        /// Marshal as JavaScript <see href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean">Boolean</see> type.
```